### DISCUSSION TITLE
Stop syncing after first file

### DISCUSSION THREAD

**Discussion by rokdd** [MobiusSync/MobiusSync#34] [Q&A] [answered]
Hello

after the first file uploading from the sync folder it stops syncing.

<img width="375" alt="20210125_155326421_iOS" src="https://user-images.githubusercontent.com/640866/105754524-7af49500-5f4a-11eb-8b39-a8de7d341a83.png">

In the log is the same message and afterwards the notice that the initial scan of folder failed.

Is there something I should / can do?

**Comment by SimonPickup** [comment]
This is probably because you have exceeded the free usage limit of 20MB. It's interesting that it hasn't prompted you differently.

Are you syncing files from your iPhone out to the world? Or downloading existing remote files onto your phone?

**Comment by rokdd** [comment]
From the iPhone to the world.. :-) The first file was 3 MB.. the error message says nothing with exceeding the limit..

**Comment by SimonPickup** [comment, marked as answer]
And are there more than 20MB total?

Have you purchase unlimited files?

There is intended to be another error explaining, but the case of iPhone to the world is not as well tested as we expected it to be less common.
